flowey/flowey_cli/src/pipeline_resolver/github_yaml/mod.rs

```diff
@@ -570,7 +570,10 @@ EOF
                         })
                         .collect()
                 },
-                r#if: gh_override_if.clone(),
+                r#if: Some(gh_override_if.clone().map_or_else(
+                    || "github.event.pull_request.draft == false".to_string(),
+                    |original| format!("{} && github.event.pull_request.draft == false", original),
```

Comment:
I think we should keep the override as a "sharp" tool, where is totally overrides any flowey-generated logic. The corresponding API method name is `gh_dangerous_override_if` after all - if you're using it, we assume you know what you're doing (which includes accounting for any require pull_request draft logic)

Reply:
> anyone using the override would "need to know"

well yes, precisely my point: anyone using the API with the word "dangerous" in it _would_ have to know to account for these edge cases. i.e: we should make sure to update our one existing caller to do the right thing in our current configuration.

the idea is that `dangerous_*` methods in flowey are very roughly analogous to `unsafe_*` "escape hatches" in regular Rust, albeit using a different key word, so as not to conflate CI safety and memory safety. They should be sharp tools that give folks a way to break free from any flowey-imposed guardrails, and do something that they know that _can_ do, even if flowey can't verify it / doesn't support it as a first-class citizen just yet.

Reply:
That's fine, I'll push a change that removes this and just adds the draft block to the override (there's only one using it right now).